MerlinAU.sh

```diff
@@ -5231,13 +5243,13 @@ _ShowMainMenu_()
       then FW_NewUpdateVersion="${REDct}NONE FOUND${NOct}"
       else FW_NewUpdateVersion="${GRNct}${FW_NewUpdateVersion}${NOct}$arrowStr"
       fi
-      printf "\n  Router's Product Name/Model ID:  $FW_RouterModelID ${padStr}(H)ide"
+      printf "\n  Router's Product Name/Model ID:  $FW_RouterModelID${padStr}(H)ide"
       printf "\n  USB-Attached Storage Connected:  $USBConnected"
       printf "\n  F/W Version Currently Installed: $FW_InstalledVersion"
-      printf "\n  F/W Update Currently Available:  $FW_NewUpdateVersion"
-      printf "\n  F/W Update Run Estimated Date:   $ExpectedFWUpdateRuntime"
+      printf "\n  F/W Update Version Available:    $FW_NewUpdateVersion"
+      printf "\n  F/W Update Estimate Run Date:    $ExpectedFWUpdateRuntime"
```

Comment:
The wording in "`F/W Update Estimate Run Date`" sounds & looks odd for a native English speaker. It should be either:

"`F/W Update Estimated Run Date`"

OR

"`F/W Update Run Estimated Date`"

IOW, "**Estimated**" is the adjective qualifying the following noun - similar to what ETA stands for.

Reply:
I can quickly change this tonight before bed just wrapping up dinner now. :) thanks for the clarification!

Reply:
Fixed in commit: https://github.com/ExtremeFiretop/MerlinAutoUpdate-Router/pull/214/commits/e49f95f4e4b4dee6d4aa06f9aefd70f0d8ae2dc2